committee-steering/sig-governance.md

```diff
@@ -0,0 +1,465 @@
+- How technical decisions are proposed and made within a SIG
+- How to ensure continued health of tests
+
+## How to use it
+
+SIGs are diverse in nature, and may have different goals based
+on what areas of the project the SIG touches.  Accordingly SIGs will
+need to adapt this template to their specific needs.  This may mean
+removing or changing roles, responsibilities, and processes.  When
+diverging from this template, please help us understand why in the
+PR description so we can improve the process and iterate on this
+document.
+
+## FAQ
+
+## Why is it so long?
+
+This document attempts to ensure the institutional knowledge
+of individuals managing SIGs today is well documented and available
+to all members in the SIG.  There are actually many responsibilities and
+processes that have developed as part of facilitating management
+of the project by a SIG comprised of a decentralized and diverse
+group of individuals.
+
+### Why so much structure?
+
+This document intentionally leans toward more structure rather than
+less.  There are 2 reasons for this:
+
+- It is easy for SIGs to remove or edit sections in this
+  template to reduce structure, whereas it is harder to make up new
+  structure from scratch
+- The SIG governance questionnaire showed a consistent preference for more
+  structure over less.  (almost all answers were to have the same or
+  more amount of structure)
+
+### Why use mailing lists for tracking roles?
+
+Using mailing lists to track roles makes it simple to contact
+the members of a SIG with a given role.  This is necessary for
+escalation and decision making to occur.  Alternatives considered
+were: OWNERs files, OWNERs alias', and GitHub teams.  However,
+none of these have a simple way to send a message to the members.
+
+SIGs may choose to have OWNERs and GitHub teams that mirror the
+mailing list membership.
+
+### Why track membership as a special role?
+
+Decision making in SIGs is lazy consensus driven, however it is
+important to define the individuals participating in lazy consensus.
+This should be the individuals responsible for the continued health
+and future direction of the area.  Individuals that do not actively
+make contributions to the area may be consulted, but are not
+ultimately responsible for decision making.
+
+---
+
+# SIG ~~Foo~~ Governance
+
+The following document outlines the governance for
+SIG ~~Foo~~.  Its goals are to define the necessary structure for
+SIG ~~Foo~~ to self govern and distribute leadership responsibilities
+within the SIG.
+
+This document is broken up into the following parts:
+
+- Roles held within the SIG and their responsibilities
+- Processes and tools used by members of the SIG to meet the
+  responsibilities of their roles
+
+## Roles and responsibilities
+
+The following section outlines the various roles governing SIG ~~Foo~~,
+their corresponding responsibilities, how each is chosen, and how
+they can be contacted.
+
+### Administrative role
```

Comment:
I feel this title should remain SIG lead. I don't see the need to change it to admin. Changing it will cause unnecessary overhead for creating new google groups. Even though this role doesn't encompass technical decision making, this person/set of people will still need to have domain experience to be able to carry out these tasks. The admin title doesn't seem to over all of the responsibilities described below. Additionally, I, as a woman, would probably not take on a role within the community that had `admin` in it. That has historically been a dangerous association for me career-wise. I can't speak for anyone else just my personal experience as an example here.

Reply:
Thank for you this insight.  I think it bothered me as well, but you articulated it well.  I changed the role name to SIG lead, and described the responsibilities as operational in nature, comparing the role to VP of operations at an organization.  Please let me know if you agree with this change.